src/main/java/DukeTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+
+    /**
+     * Method to test the file reading capabilities of the storage class.
+     */
+    @Test
+    public void Test1() {
```

Comment:
Perhaps a more descriptive name for the test could be used here?

Reply:
You can refer to the naming conventions for tests here https://se-education.org/guides/conventions/java/index.html under the "Naming" section!